.ci/atime/tests.R

```diff
@@ -166,5 +166,25 @@ test.list <- atime::atime_test_list(
     Slow = "a01f00f7438daf4612280d6886e6929fa8c8f76e", # Parent of the first commit (https://github.com/Rdatatable/data.table/commit/fc0c1e76408c34a8482f16f7421d262c7f1bde32) in the PR (https://github.com/Rdatatable/data.table/pull/6296/commits) that fixes the issue
     Fast = "f248bbe6d1204dfc8def62328788eaadcc8e17a1"), # Merge commit of the PR (https://github.com/Rdatatable/data.table/pull/6296) that fixes the issue
 
+  # Issue reported in: https://github.com/Rdatatable/data.table/issues/3735
+  # To be fixed in: https://github.com/Rdatatable/data.table/pull/4488
+  "DT[i] improved in #4488" = atime::atime_test(
+    N = 10^seq(1, 20),
+    setup = {
+      allIterations <- data.frame(v1 = runif(N), v2 = runif(N))
+      DoSomething <- function(row) {
+      someCalculation <- row[["v1"]] + 1
+      }
+      allIteration_dt <- as.data.table(allIterations)
```

Comment:
what's the point of this line? it's not being used anywhere inside the evaluated expression, noh? (`expr` is only using `allIterations`)